pkg/provider/gke/gke.go

```diff
@@ -43,9 +42,15 @@ import (
 )
 
 // New is the GKE constructor.
-func New() *GKE {
+func New(dr provider.DeploymentResource) *GKE {
```

Comment:
There's a bug here, `dr` get passed when it's corresponding flags not even parsed.